integration-tests/utils/integration-utils.js

```diff
@@ -73,7 +73,7 @@ const getTableColIndex = async (page, { tableName, columnHeader }) => {
 const getTableCellByContents = async (page, { tableName, columnHeader, textContains, isDescendant = false }) => {
   const colIndex = await getTableColIndex(page, { tableName, columnHeader })
   const baseXpath = `//*[@role="table" and @aria-label="${tableName}"]//*[@role="row"]//*[@role="cell" and @aria-colindex = "${colIndex}"]`
-  const xpath = `${baseXpath}${isDescendant ? '//*' : ''}[contains(normalize-space(.),"${textContains}")]`
+  const xpath = `${baseXpath}${isDescendant ? '//*' : ''}[text() = "${textContains}"]`
```

Comment:
Could you explain the reasoning behind the change above?

Reply:
Discovered in alpha test that it picked up `patient13` before `patient1` and re-testing with `[text() = "${textContains}"]` didn't create new errors.

Reply:
Do we need to care about the ordering of content? If not, can we do the comparison in a way where order isn't important (e.g. as sets).